File: Form1.cs
using System;
using System.Linq;
using System.Windows.Forms;

namespace classTest1
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            VisualStyleManager.AddStyle("ButtonStyle1", button1);

            //https://stackoverflow.com/questions/4630391/get-all-controls-of-a-specific-type
            foreach (var btn in this.Controls.OfType<Button>()) 
            {
                if (btn != button1)
                {
                    VisualStyleManager.ApplyStyle("ButtonStyle1", btn);
                }
            }
        }
    }
}

File: Form1.vb
Public Class Form1
    Private Sub button1_Click(sender As Object, e As EventArgs) Handles button1.Click
        Dim vsm As VisualStyleManager = New VisualStyleManager()
        vsm.AddStyle("ButtonStyle1", button1)

        For Each btn As Control In Me.Controls.OfType(Of Button)
            If btn IsNot button1 Then
                vsm.ApplyStyle("ButtonStyle1", btn)
            End If
        Next



    End Sub
End Class
